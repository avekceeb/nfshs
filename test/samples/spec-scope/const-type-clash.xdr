enum A { X = 1; Y = 2 };
const B = 2;
struct A { int C; };