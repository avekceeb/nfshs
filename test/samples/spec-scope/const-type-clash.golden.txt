test/samples/spec-scope/const-type-clash.xdr:1:15:
unexpected ';'
expecting ',', '}', or the rest of integer
